### ISSUE TITLE
default parameters to reproduce all the dataset mentioned in the paper

### ISSUE THREAD

**Issue by he010103** [TAMU-VITA/ABD-Net#8]
@hsfzxjy Can you provide the parameters to reproduce the results in different datasets, such as market1501, CUHK03, etc. ? Thanks!

**Comment by hsfzxjy** [comment]
You could use the command mentioned in [our training guide](https://github.com/TAMU-VITA/ABD-Net/blob/master/README_Training_and_Testing_Guides.md), with `-s` and `-t` substituded to corresponding dataset name. Market-1501, DukeMTMC-ReID and MSMT17 share the same training hyper-parameters. As of CUHK03, we are still doing the experiments, and will publish details in external journal work.

**Comment by he010103** [comment]
@hsfzxjy I use the command in the [https://github.com/TAMU-VITA/ABD-Net/blob/master/README_Training_and_Testing_Guides.md](training guide)
`
srun --mpi=pmi2 -p VI_UC_1080TI --gres=gpu:2  -u python train.py -s market1501 -t market1501 \
    --flip-eval --eval-freq 10 \
    --label-smooth \
    --criterion htri \
    --lambda-htri 0.1  \
    --data-augment crop,random-erase \
    --margin 1.2 \
    --train-batch-size 48 \
    --height 384 \
    --width 128 \
    --optim adam \
    --lr 0.0003 \
    --stepsize 20 40 \
    --gpu-devices 0,1 \
    --max-epoch 80 \
    --save-dir log \
    --arch resnet50 \
    --use-of \
    --abd-dan cam pam \
    --abd-np 2 \
    --shallow-cam \
    --use-ow
`
But after training, the performance is poor:
`
Model size: 69.175 M
Loaded pretrained weights from 'log/checkpoint_ep80.pth.tar'
Evaluate only
Evaluating market1501 ...
# Using Flip Eval
Extracted features for query set, obtained 3368-by-3072 matrix
Extracted features for gallery set, obtained 15913-by-3072 matrix
==> BatchTime(s)/BatchSize(img): 0.190/100
Computing CMC and mAP
Results ----------
mAP: 72.77%
CMC curve
Rank-1  : 88.15%
Rank-5  : 95.01%
Rank-10 : 96.56%
Rank-20 : 97.71%
------------------
`

**Comment by he010103** [comment]
> You could use the command mentioned in [our training guide](https://github.com/TAMU-VITA/ABD-Net/blob/master/README_Training_and_Testing_Guides.md), with `-s` and `-t` substituded to corresponding dataset name. Market-1501, DukeMTMC-ReID and MSMT17 share the same training hyper-parameters. As of CUHK03, we are still doing the experiments, and will publish details in external journal work.
